enos/modules/build_boundary_docker_crt/main.tf

```diff
@@ -23,10 +23,22 @@ variable "edition" {
   default = "oss"
 }
 
+locals {
+  docker_image_filename      = basename(var.path)
+  filename_parts             = split("_", replace(local.docker_image_filename, ".docker.tar", ""))
+  arch                       = local.filename_parts[3]
+  version_and_commit         = join("_", slice(local.filename_parts, 4, length(local.filename_parts)))
+  boundary_docker_image_name = "boundary/default/linux/${local.arch}:${local.version_and_commit}"
+}
```

Comment:
This approach feels fairly brittle. It also doesn't work for `boundary-enterprise` due to the name being different.

Can we not examine the output of `docker load -i` and grab the info from the last line?
```
❯ docker load -i ~/Downloads/boundary-enterprise_default_linux_amd64_0.20.0+ent_659b52ac6d63e6d20eadde1f8106f9ddfbe577db.docker.tar 
Loaded image: hashicorp/boundary-enterprise:0.20.0-ent
Loaded image: hashicorp/boundary-enterprise:0.20.0-ent_659b52ac6d63e6d20eadde1f8106f9ddfbe577db
Loaded image: public.ecr.aws/hashicorp/boundary-enterprise:0.20.0-ent
Loaded image: boundary-enterprise/default/linux/amd64:0.20.0-ent_659b52ac6d63e6d20eadde1f8106f9ddfbe577db
```

You should be able to access the output from `enos_local_exec.load_docker_image.stdout`

Reply:
Also, make sure to keep an eye on the CI checks. As you can see, the docker e2e tests are failing.